techniques/failures/F37.html

```diff
@@ -56,7 +56,11 @@ <h3></h3>
          </ol>
       </section>
       <section class="results"><h3>Expected Results</h3>
-         <p> If check #3 is false, then this failure condition applies and content
-                            fails the Success Criterion.</p>
+         <p>If check #3 is false, then this failure condition applies and content fails the Success Criterion.</p>
       </section>
-   </section><section id="related"><h2>Related Techniques</h2></section><section id="resources"><h2>Resources</h2></section></body></html>
\ No newline at end of file
+      <div class="note">Note that in the case of a set of radio buttons, adding an indication or warning
+         that selecting a radio button will result in a change of context will pass the requirements of 3.2.2 – however,
+         this scenario would then still fail 2.1.1 Keyboard, since it's not possible (in current user agents) for a user to
```

Comment:
An attempt to address Detlev's comment
```suggestion
         this scenario would then likely fail 2.1.1 Keyboard, since it's not possible (in current user agents) for a user to
```